.github/workflows/build-make-check.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build with warnings and make check"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  makecheck:
+    runs-on: ubuntu-latest
```

Comment:
Besides Ubuntu, we want to make sure to build it in Fedora, we can start with fedora:latest and add more later when we migrate and optimize current ci/run.

Reply:
Our current https://github.com/SSSD/sssd/blob/master/.github/actions/install-dependencies/action.yml needs some changes to work inside containers (ex: `lsb_release` command not available). Should `distro.sh` be extended to work inside a container, or should I just add some additional steps to install missing commands inside `action.yml` ? Or do you already have some script that can be re-used from your work in https://github.com/SSSD/sssd-ci-containers/ ?

Reply:
IMHO we can switch to sourcing /etc/os-release instead of using lsb_release.

Reply:
You can also use https://quay.io/repository/sssd/ci-client-devel which has dependencies preinstalled, however it does not really solve the problem - in case a pull requests adds new dependency, you still need to run install-dependencies in order to pick added packages before they are added to the container.